albam/blender_ui/tools.py

```diff
@@ -81,12 +81,12 @@ def draw(self, context):
 @blender_registry.register_blender_type
 class ALBAM_PT_VGMerger(bpy.types.Panel):
     '''UI Tool subpanel in Mesh Object Data'''
-    bl_label = "Vertex Groups Merger (Albam)"
+    bl_label = "Vertex Groups Merger"
     bl_idname = "ALBAM_PT_VGMerger"
-    bl_space_type = 'PROPERTIES'
-    bl_region_type = 'WINDOW'
-    bl_context = "data"
-    CONTEXT_ITEM_NAME = "mesh"
+    bl_category = "Albam [Beta]"
```

Comment:
Instead of creating a new panel, you can add it as a sub-panel of the tools panel by setting `bl_parent_id = ALBAM_PT_ToolsPanel`. Wouldn't it be cleaner?